lib/trie/trie.go

```diff
@@ -1,7 +1,7 @@
 package trie
 
 import (
-	"boscoin.io/sebak/lib/common"
+	sebakcommon "boscoin.io/sebak/lib/common"
```

Comment:
?

Reply:
common is duplicated with go-ethereum/common

Reply:
Then rename the foreign common, not our own.